allennlp/models/basic_classifier.py

```diff
@@ -68,10 +68,12 @@ def __init__(self,
         else:
             self._dropout = None
 
+        self._label_namespace = label_namespace
+
         if num_labels:
             self._num_labels = num_labels
         else:
-            self._num_labels = vocab.get_vocab_size(namespace=label_namespace)
+            self._num_labels = vocab.get_vocab_size(namespace=self._label_namespace)
```

Comment:
I don't think there's any need to do this, the original `label_namespace` is still in scope here?